lib/bindings/HTTPBinding.js

```diff
@@ -117,9 +117,8 @@ function executeCommand(apiKey, device, serializedPayload, callback) {
             callback(new errors.HTTPCommandResponseError(response.statusCode, error));
         } else {
             process.nextTick(commandHandler.updateCommand.bind(null, apiKey, device, body, callback));
+            callback();
```

Comment:
I think you have to remove it from here too since already included into the previous function call as the callback.